src/lib/coda_numbers/nat.mli

```diff
@@ -1,8 +1,4 @@
-module type S_unchecked = Intf.S_unchecked
-
-module type S_checked = Intf.S_checked
-
-module type S = Intf.S
```

Comment:
we did intentionally hide the coercion to and from uint32 and uint64 so nothing bad could happen, but we do need to expose it to access the data from graphql, so this seems fine